### PR TITLE
Update oscap plugin changes for sat6.2

### PR DESCRIPTION
**Pull request by kbidarkar**

### CODE REVIEW

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -534,21 +534,29 @@ def setup_abrt():
...
+        ]
+        for package in packages:
+            run('yum install -y {0}'.format(package))
+
+        for daemon in ('foreman', 'httpd', 'foreman-proxy'):
+            manage_daemon('restart', daemon)
+    else:
+       # Run foreman-installer to enable oscap plugin
+       foreman-installer --enable-foreman-plugin-openscap
+       foreman-installer --enable-foreman-proxy-plugin-openscap
```
Comment (kbidarkar) [inline automation_tools/__init__.py]: Oops! fixing it.

File: automation_tools/__init__.py [hunk 2]
```diff
@@ -534,21 +534,29 @@ def setup_abrt():
 def setup_oscap():
     """Task to setup oscap on foreman."""
     # Check if ruby193-rubygem-foreman_openscap package is available
-    result = run('yum list ruby193-rubygem-foreman_openscap', quiet=True)
+    if os.environ.get('SATELLITE_VERSION') == '6.2':
+	result = run('yum list tfm-rubygem-foreman_openscap', quiet=True)
```
Comment (sthirugn) [inline automation_tools/__init__.py]: add an other space here

File: automation_tools/__init__.py [hunk 3]
```diff
@@ -534,21 +534,29 @@ def setup_abrt():
...
-    for daemon in ('foreman', 'httpd', 'foreman-proxy'):
-        manage_daemon('restart', daemon)
+    if os.environ.get('SATELLITE_VERSION') == '6.1':
+        # Install required packages for the installation
+        packages = [
+            'rubygem-smart_proxy_openscap',
+            'ruby193-rubygem-foreman_openscap'
+        ]
+        for package in packages:
+            run('yum install -y {0}'.format(package))
```
Comment (sthirugn) [inline automation_tools/__init__.py]: you can do this https://github.com/SatelliteQE/automation-tools/pull/322/files#diff-a692ae115cc45cc7627d4545e9b4c294R568

File: automation_tools/__init__.py [hunk 4]
```diff
@@ -534,21 +534,29 @@ def setup_abrt():
...
+
+        for daemon in ('foreman', 'httpd', 'foreman-proxy'):
+            manage_daemon('restart', daemon)
+    else:
+       # Run foreman-installer to enable oscap plugin
+       run('foreman-installer --enable-foreman-plugin-openscap '
```
Comment (sthirugn) [inline automation_tools/__init__.py]: looks like this needs indentation fix too

File: automation_tools/__init__.py (hunk 2 above)
Comment (kbidarkar) [inline automation_tools/__init__.py]: weird indentation at my end appears to be right

File: automation_tools/__init__.py (hunk 3 above)
Comment (kbidarkar) [inline automation_tools/__init__.py]: Let's leave this as it is, no changes here. Just indentation changed.

**Comment by sthirugn** [comment]
ACK

**Pull request merged**